Makefile

```diff
@@ -1444,6 +1449,15 @@ ui-maintainer-clean: ## Like clean, but also remove installed dependencies
 ui-maintainer-clean: ui-clean
 	rm -rf pkg/ui/node_modules pkg/ui/workspaces/db-console/node_modules pkg/ui/yarn.installed pkg/ui/workspaces/cluster-ui/node_modules
 
+pkg/security/securitytest/embedded.go: bin/.bootstrap $(shell find pkg/security/securitytest/test_certs -type f -not -name README.md -not -name regenerate.sh)
```

Comment:
Do you mind explaining why these specific files are special cased? Are there other go:generate ones that aren't? What about the stringer files? See top-level comment -- do we plan to .gitignore other kinds of generated code shortly?